src/models/queryRangeGroupBy.js

```diff
@@ -0,0 +1,23 @@
+class QueryRangeGroupBy {
+    /**
+     * [constructor description]
+     * @param  {String} field  [description]
+     * @param  {Array}  ranges [description]
+     * @return {QueryRangeGroupBy Class}        [description]
+     */
+    constructor(field, ranges = []) {
```

Comment:
ranges is a mandatory for QueryRangeGroupBy see specs. Could be good to test it.